kolibri_instant_schools_plugin/assets/src/views/vf-accent-button/index.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+
+  <k-button
+    :text="text"
+    :primary="primary"
+    :disabled="disabled"
+    :type="type"
+    @click="$emit('click')"
+    class="vf-accent-button"
+  />
+
+</template>
+
+
+<script>
+
+  import kButton from 'kolibri.coreVue.components.kButton';
+
+  export default {
+    name: 'vfAccentButton',
+    props: ['text', 'primary', 'disabled', 'type'],
+    components: {
+      kButton,
+    },
+  };
+
+</script>
+
+
+<style scoped lang="stylus">
+
+  @require '~kolibri.styles.definitions'
+
+  .vf-accent-button
+    background-color: $core-accent-color
+    &:hover
+      background-color: #f44336 // ported red color
+
+</style>
```

Comment:
recommend configuring your editor to insert [trailing newlines](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) :)